compose/gettingstarted.md

```diff
@@ -158,13 +158,13 @@ Hub registry.
     Compose pulls a Redis image, builds an image for your code, and starts the
     services you defined. In this case, the code is statically copied into the image at build time.
 
-2.  Enter `http://0.0.0.0:5000/` in a browser to see the application running.
+2.  Enter http://localhost:5000/ in a browser to see the application running.
 
     If you're using Docker natively on Linux, Docker Desktop for Mac, or Docker Desktop for
     Windows, then the web app should now be listening on port 5000 on your
-    Docker daemon host. Point your web browser to `http://localhost:5000` to
+    Docker daemon host. Point your web browser to http://localhost:5000 to
     find the `Hello World` message. If this doesn't resolve, you can also try
-    `http://0.0.0.0:5000`.
+    http://127.0.0.1:5000 or http://0.0.0.0:5000.
```

Comment:
I don't think we need `127.0.0.1` here as it's the same as localhost.

Reply:
I've noticed that sometimes Docker will only bind to `localhost` or `127.0.0.1`, not both for some reason. However, `0.0.0.0` would never be correct, so I'll remove that.